skrifa/src/outline/autohint/outline.rs

```diff
@@ -697,4 +731,56 @@ mod tests {
         outline.fill(&glyph, Default::default()).unwrap();
         outline
     }
+
+    /// Ensure that autohint outline to path conversion produces the same
+    /// output as our other scalers for all formats and path styles including
+    /// edge cases like mostly off-curve glyphs
+    #[test]
+    fn outline_to_path_conversion() {
+        compare_outlines("MOSTLY_OFF_CURVE", font_test_data::MOSTLY_OFF_CURVE);
+        compare_outlines("STARTING_OFF_CURVE", font_test_data::STARTING_OFF_CURVE);
+        compare_outlines("CUBIC_GLYF", font_test_data::CUBIC_GLYF);
+        compare_outlines("VAZIRMATN_VAR", font_test_data::VAZIRMATN_VAR);
+        compare_outlines("CANTARELL_VF_TRIMMED", font_test_data::CANTARELL_VF_TRIMMED);
+        compare_outlines(
+            "NOTO_SERIF_DISPLAY_TRIMMED",
+            font_test_data::NOTO_SERIF_DISPLAY_TRIMMED,
+        );
+    }
```

Comment:
This is many distinct tests hiding in one. It's a bad pattern that I've seen cause inconvenience many times because it denies us the ability to see which tests fail by just bailing on first failure.

Make individual `#[test]` entrypoints instead.

Reply:
Done